config/environments/production.rb

```diff
@@ -76,4 +76,5 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+  config.action_mailer.default_url_options = { host: 'afternoon-fortress-49236.herokuapp.com' }
```

Comment:
Yes, it is deployed to Heroku with the updates from this checkpoint and appears to be working.